manifests/charts/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -175,6 +175,8 @@ spec:
           {{- end }}
           - name: local-certs
             mountPath: /var/run/secrets/istio-dns
+          - name: remote-token
+            mountPath: /var/run/secrets/remote-token
```

Comment:
optional?

Reply:
also should this be guarded with externalIstiod flag?

Reply:
sure, will guard it once externalIstio flag is landed